test/integration/targets/lineinfile/tasks/main.yml

```diff
@@ -253,6 +253,16 @@
     that:
       - "result.stat.checksum == 'ab56c210ea82839a54487464800fed4878cb2608'"
 
+###################################################################
+# 1st literal tests
+
+- import_tasks: test_literal.yml
+  vars:
+    stage: first
```

Comment:
Just make two tasks files in the appropriate order rather than having to user `vars` and a block with a conditional.